terraform/modules/single_instance/variable.tf

```
@@ -23,6 +25,8 @@ variable "vsc_enabled" {
 variable "project_name" {
   default = "default"
 }
+variable "floatingip_address" {
+}
```

This should be optional:
```suggestion
default = null
}
```